### PR TITLE
DOC: Miscellaneous community doc improvements

### PR DESCRIPTION
**Pull request by jhlegarreta**
- DOC: Add a note to the `README` about issues vs. questions
- DOC: Add a note to the first interaction workflow about its nature
- DOC: Refer to ITK's discussion space in a consistent manner
- DOC: Reword the copyright holder sentence in the code of conduct
- DOC: Link the Creative Commons license page in the code of conduct

## PR Checklist
- [X] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [X] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)

### CODE REVIEW

File: .github/workflows/first-interaction.yml [hunk 1]
```diff
@@ -22,10 +22,17 @@ jobs:
 
           We are glad you are here and appreciate your contribution. Please keep in mind our [community participation guidelines](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CODE_OF_CONDUCT.md). 📜
           Also, please check [existing open issues](https://github.com/InsightSoftwareConsortium/ITK/issues) and consider discussion on the [ITK Discourse](https://discourse.itk.org). 📖
+
+          This is an automatic message. Hopefully, this is not all that will happen ! But allow for some time to the ITK
+          community to be able to read the issue and comment on it.
```
Comment (thewtex) [inline .github/workflows/first-interaction.yml:27]: An adjustment that could be more succinct and positive.

> Hopefully, this is not all that will happen ! But allow for some time to the ITK￼community to be able to read the issue and comment on it.

to

> Allow for time for the ITK community to be able to read the issue and comment on it.

**Review by thewtex — COMMENTED** [review]
Thanks @jhlegarreta !

File: .github/workflows/first-interaction.yml (hunk 1 above)
Comment (jhlegarreta) [reply .github/workflows/first-interaction.yml]: Done in [b774fd5](https://github.com/InsightSoftwareConsortium/ITK/pull/4133/commits/b774fd5358112c019466b95db72f87f97b2f6671).

**Review by thewtex — APPROVED** [review]

**Comment by thewtex** [comment]
thanks @jhlegarreta !

**Comment by jhlegarreta** [comment]
ghostflow-check-master failure looks a hiccup to me. Cannot retrigger it manually.

**Comment by thewtex** [comment]
Let's ignore the ghostflow-check-master.

**Pull request merged**